src/main/flight/imu.c

```diff
@@ -87,14 +86,18 @@ static float throttleAngleScale;
 static float fc_acc;
 static float smallAngleCosZ = 0;
 
-static float magneticDeclination = 0.0f;       // calculated at startup from config
```

Comment:
@adrianmiriuta : Sorry, you are right ... `magneticDeclination` configuration was removed in past, it is always zero / unfinished ...